contracts/RemBadgerDripper.sol

```diff
@@ -0,0 +1,75 @@
+// contracts/RemBadgerDripper.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/finance/VestingWallet.sol";
+
+contract RemBadgerDripper is VestingWallet {
+    mapping(address => uint256) private _erc20Released;
+    address private immutable _controller;
+    address private immutable _governance;
```

Comment:
```suggestion
    address public constant controller = 0x86cbD0ce0c087b482782c181dA8d191De18C8275;
    address public constant governance = 0xB65cef03b9B89f99517643226d76e286ee999e77;
```
You can keep the variables as public constants if they're going to be hardcoded. Constants are inlined during compilation and don't take a storage slot. Will save some `SLOAD` gas when the variables are accessed. The `public` automatically adds a getter.

Reply:
i assume i can then also remove these lines from the constructor?
```
        _controller = 0x86cbD0ce0c087b482782c181dA8d191De18C8275; //techops
        _governance = 0xB65cef03b9B89f99517643226d76e286ee999e77; //dev
```

Reply:
Yep. Would also have to change references to `controller()`/`governance()` to `controller/governance`